test/unit/transloadit/test_smart_cdn_node_parity.rb

```diff
@@ -0,0 +1,112 @@
+require "test_helper"
+require "json"
+require "open3"
+
+describe Transloadit do
+  # Only run these tests if we're in CI or explicitly requested
+  if ENV["TEST_NODE_PARITY"] == "1"
+    before do
+      @transloadit = Transloadit.new(key: "my-key", secret: "my-secret")
+      @workspace = "my-app"
+      @template = "test-smart-cdn"
+      @input = "inputs/prinsengracht.jpg"
+      @expire_at = 1732550672867
+
+      # Fail if tsx is not available but was explicitly requested
+      unless system("which tsx > /dev/null 2>&1")
+        raise "tsx is required for node parity tests. Please install with: npm install -g tsx"
+      end
+    end
+
+    def run_node_script(params)
+      script_path = File.expand_path("./node-smartcdn-sig", __dir__)
+      json_input = JSON.dump(params)
+      stdout, stderr, status = Open3.capture3("tsx #{script_path}", stdin_data: json_input)
+      raise "Node script failed: #{stderr}" unless status.success?
+      stdout.strip
+    end
+
+    describe "#signed_smart_cdn_url" do
+      it "generates urls that match node implementation" do
+        params = {
+          workspace: @workspace,
+          template: @template,
+          input: @input,
+          expire_at_ms: @expire_at
+        }
+
+        url = @transloadit.signed_smart_cdn_url(**params)
+        node_url = run_node_script(params)
+        assert_equal node_url, url
```

Comment:
These test cases only check if the Ruby implementation matches the Node.js script in this repository. We should also compare the generated URLs against static strings here to ensure they are what we expect them to be:

```suggestion
        assert_equal node_url, url
        assert_equal "https://....", url
```

This makes the tests easier to understand since you directly see what the expected output is. In addition, we test the Node.js script to ensure it doesn't produce total garbage :)

Reply:
I changed it to have just one test file, that tests both against node as a hardcoded string